packages/components/pie-input/src/index.ts

```diff
@@ -122,21 +124,26 @@ export class PieInput extends FormControlMixin(RtlMixin(LitElement)) implements
             type, value, name, pattern, minlength, maxlength, autocomplete, placeholder, autoFocus, inputmode, readonly,
         } = this;
 
-        return html`<input
-            type=${ifDefined(type)}
-            .value=${live(value)}
-            name=${ifDefined(name)}
-            pattern=${ifDefined(pattern)}
-            minlength=${ifDefined(minlength)}
-            maxlength=${ifDefined(maxlength)}
-            autocomplete=${ifDefined(autocomplete)}
-            ?autofocus=${autoFocus}
-            inputmode=${ifDefined(inputmode)}
-            placeholder=${ifDefined(placeholder)}
-            ?readonly=${readonly}
-            @input=${this.handleInput}
-            @change=${this.handleChange}
-            data-test-id="pie-input">`;
+        return html`
+            <div>
```

Comment:
Interesting if it is possible to use empty tags instead of actual html elements here similar like it is possible in react `<></>`

Reply:
@dandel10n I think that's only applicable for JSX, and not part of the html standards

Reply:
Even if it were possible, I need a wrapper element for the styling later on 😃